packages/core-plugin-v1/src/index.ts

```diff
@@ -58,3 +58,28 @@ export * from './runtime.ts';
 // - database
 // - knowledge / memory
 // - relationships
+
+// Export main runtime compatibility class
+export { CompatAgentRuntime } from './runtime';
+
+// Export types
+export * from './types';
+
+// Export translators
+export { translateV1MemoryToV2, translateV2MemoryToV1 } from './translators/memory-translator.ts';
```

Comment:
I don't see any reason to export any function that the v1 core doesn't already export. Any code changes to a v1 plugin should me to upgrade it to use core-plugin-v2 or greater